docs/_content/ontwikkelaars/handleidingen-en-tutorials/large-files.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Large files upload tutorial"
+weight: 100
+---
+
+In this tutorial we explore the ways to upload files to DRC depending on the file size. 
+
+The most relevant component for this tutorial:
+
+* DRC: for creating documents and uploading files
```

Comment:
Might be nitpicking, but you could maybe add a link to the openAPI schema of the DRC (https://documenten-api.vng.cloud/api/v1/schema/) here as a reference

Reply:
The link to DRC OAS is added